akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowDispatcherSpec.scala

```diff
@@ -4,11 +4,13 @@
 
 package akka.stream.scaladsl
 
-import akka.stream.testkit.StreamSpec
-import akka.testkit.TestProbe
 import akka.stream.ActorMaterializer
 import akka.stream.ActorMaterializerSettings
+import akka.stream.testkit.StreamSpec
+import akka.testkit.TestProbe
+import com.github.ghik.silencer.silent
 
+@silent("deprecated")
```

Comment:
Not updated?

Reply:
Same here, it actually verifies that the setting changes the dispatcher for a flow